rate-providers/asETHRateProvider.md

```diff
@@ -0,0 +1,65 @@
+# Rate Provider: `Aspida asETH Rate Provider`
+
+## Details
+- Reviewed by: @wei3erHase
+- Checked by: @\<GitHub handle of secondary reviewer\>
+- Deployed at:
+    - `saETH(Proxy)` [eth:0xF1617882A71467534D14EEe865922de1395c9E89](https://etherscan.io/address/0xF1617882A71467534D14EEe865922de1395c9E89)
+    - `saETH(Implementation)` [eth:0xD9F64Ee3DD6F552c1BcfC8862dbD130bc6697a66](https://etherscan.io/address/0xD9F64Ee3DD6F552c1BcfC8862dbD130bc6697a66)
+    - `aETH(Proxy)` [eth:0xFC87753Df5Ef5C368b5FBA8D4C5043b77e8C5b39](https://etherscan.io/address/0xFC87753Df5Ef5C368b5FBA8D4C5043b77e8C5b39)
+    - `aETH(Implementation)` [eth:0x5f898DC62d699ecBeD578E4A9bEf46009EA8424b](https://etherscan.io/address/0x5f898DC62d699ecBeD578E4A9bEf46009EA8424b)
```

Comment:
```suggestion
    - [ethereum:0x1aCB59d7c5D23C0310451bcd7bA5AE46d18c108C](https://etherscan.io/address/0x1aCB59d7c5D23C0310451bcd7bA5AE46d18c108C#code)
```

Reply:
This section only contains the addresses of the deployed rate Prviders. Upgradeable contracts are mentioned in the registry and the full-text review below.